smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/OperationStep.kt

```diff
@@ -10,10 +10,10 @@ import software.amazon.smithy.swift.codegen.swiftmodules.ClientRuntimeTypes
 
 abstract class OperationStep(outputType: Symbol, outputErrorType: Symbol) {
     abstract val inputType: Symbol
-    val outputType: Symbol = Symbol
-        .builder()
+    val outputType: Symbol = Symbol.builder()
         .name("${ClientRuntimeTypes.Middleware.OperationOutput}<$outputType>")
-        .addDependency(SwiftDependency.CLIENT_RUNTIME)
+        .addReference(ClientRuntimeTypes.Middleware.OperationOutput)
+        .addReference(outputType)
```

Comment:
`.addReference()` allows a symbol to import another symbol along with itself.  This is useful for when you want to have a symbol that has another in its definition, such as collection types.